ui/src/app/metadata/named-references/NamedReference.ts

```diff
@@ -1,25 +1,48 @@
 import {MetadataType} from "../rsd-metadata.enum"
 import {Meta} from "@angular/platform-browser"
+import {IJobCode, JobCodeLevel} from "../job-codes/Jobcode";
```

Comment:
Line 2 and 3 are unused imports.

Reply:
oops, forgot to delete those imports after paste code.